Tests/SwiftParserTest/translated/InvalidTests.swift

```diff
@@ -283,7 +283,7 @@ final class InvalidTests: ParserTestCase {
           message: "expected ')' in function type",
           fixIts: ["insert ')'"]
         ),
-        DiagnosticSpec(locationMarker: "3️⃣", message: "expected return type in function type", fixIts: ["insert return type"]),
+        DiagnosticSpec(locationMarker: "3️⃣", message: "expected return type in return clause", fixIts: ["insert return type"]),
```

Comment:
Not sure if all those cases are effects of that I added `return clause` as `nameForDiagnostics`